mnemosyne/diagnose.py

```
@@ -132,6 +132,18 @@ def _memory_orphan_diagnostics(conn) -> Dict[str, int]:
     return diagnostics
 
 
+
+
+def _sqlite_integrity_diagnostics(conn) -> Dict[str, str]:
+    """Return PII-safe SQLite integrity diagnostics."""
+    try:
+        row = conn.execute("PRAGMA quick_check").fetchone()
+        result = str(row[0]) if row else "unknown"
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)